docs/sql/got-tutorial/sql-crud.md

```diff
@@ -0,0 +1,154 @@
+---
+sidebar_position: 10
+title: Part 1 - Basic Document Tasks
+---
+
+You can perform basic document tasks such as create, read, update, and delete (CRUD) on documents in a collection. This portion of the tutorial guides you through those tasks.
+We will firs review the syntax of CRUD commands and then use SDKs to send queries to Macrometa platform.
+
+## Create the Characters Collection
+
+Before we can insert documents with SQL, we need a place to put them in: a collection.
+
+For this tutorial, [Create a Document Store collection](../../collections/documents/create-document-store.md) in the console. For more information about collections, refer to [Collections](../../collections/index.md).
+
+1. [Log in to your Macrometa account](https://auth.paas.macrometa.io/).
+2. Click **Collections**.
+3. Click **New Collection**.
+4. Click **Document Store**.
+5. Name the collection **categories** and then click **Create**.
+
+### Add One Document to the Collection
+
+Add one document to the collection with a query.
```

Comment:
Where do I enter this query? In my terminal? In the API docs in the app? In the Editor tab in Query Workers?

Reply:
When enabled this will work from our console query workers, which is not possible atm. But I placed these commands as a placeholder.